src/teleop/gui/src/components/CameraInfo.vue

```diff
@@ -69,17 +79,17 @@ export default {
 </script>
 
 <style scoped>
-.box {
-  border-radius: 5px;
-  padding: 10px;
-  border: 1px solid black;
-}
-.wrap {
+.wrapper {
+  /* wrap acts weird with flex from parent... */
```

Comment:
Does this mean it acts weird if you name it wrap or that the class itself acts weird currently?